### PR TITLE
Feature/414 parachain details screen (validators and non-voters)

### PR DESCRIPTION
**Pull request by ishrak-imam**
### Description
Validators and non voters list in the parachain details screen.

### CODE REVIEW

File: src/screen/Parachains/ParachainDetailScreen.tsx [hunk 1]
```diff
@@ -1,102 +1,240 @@
...
+      .replace(/_/g, '')
+      .split('')
+      .map((c, index) => (c === '0' ? validators[index] : null))
+      .filter((v, index): v is AccountId => !!v && index < validators.length);
+  }
+
+  return list;
+}
+
+// function useParachainHomepage(id: ParaId) {
```
Comment (geastwood) [inline src/screen/Parachains/ParachainDetailScreen.tsx]: Do we need this code_

**Review by geastwood — COMMENTED** [review]
Good work!

File: src/screen/Parachains/ParachainDetailScreen.tsx (hunk 1 above)
Comment (ishrak-imam) [reply src/screen/Parachains/ParachainDetailScreen.tsx]: No, not needed.

File: src/screen/Parachains/ParachainDetailScreen.tsx [hunk 2]
```diff
@@ -1,102 +1,231 @@
...
+        contentContainerStyle={styles.content}
+        stickySectionHeadersEnabled={false}
+        sections={sections}
+        renderItem={({item}) => <>{item && <Validator accountId={item} />}</>}
+        renderSectionHeader={({section: {title}}) => (
+          <Text category={'s1'} style={styles.header}>
+            {title}
+          </Text>
+        )}
+        keyExtractor={(item) => (item ? item?.toString() : '')}
```
Comment (silva-fj) [inline src/screen/Parachains/ParachainDetailScreen.tsx]: What's the type of `item` here? You shouldn’t have to do this

File: src/screen/Parachains/ParachainDetailScreen.tsx [hunk 3]
```diff
@@ -111,4 +240,5 @@ const styles = StyleSheet.create({
   text: {
     fontFamily: monofontFamily,
   },
+  button: {},
```
Comment (silva-fj) [inline src/screen/Parachains/ParachainDetailScreen.tsx:245]: Remove?

**Review by silva-fj — DISMISSED** [review]

File: src/screen/Parachains/ParachainDetailScreen.tsx (hunk 2 above)
Comment (ishrak-imam) [reply src/screen/Parachains/ParachainDetailScreen.tsx]: `AccountId | undefined` But I think it is fixable. I'll look into it.

File: src/screen/Parachains/ParachainDetailScreen.tsx [hunk 4]
```diff
@@ -1,102 +1,231 @@
...
+    [parachainValidators, parachainInfo],
+  );
+
+  const endpoints = useParaEndpoints(id as unknown as ParaId);
+  const homepage = endpoints?.length ? endpoints[0]?.homepage : undefined;
+
+  const sections = [
+    {
+      title: `Val. Group ${validatorInfo?.groupIndex.toNumber() || ''} (${validatorInfo?.validators?.length || 0})`,
+      data: validatorInfo?.validators || [],
```
Comment (ahoseinian) [inline src/screen/Parachains/ParachainDetailScreen.tsx:94]: notEmpty is a function in utils if you would like to get rid of the `undefined`s in the array
```suggestion
      data: validatorInfo?.validators.filter(notEmpty) || [],
```

**Review by ahoseinian — DISMISSED** [review]

**Review by ahoseinian — DISMISSED** [review]

**Review by ahoseinian — APPROVED** [review]

**Pull request merged**